tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2055,471 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
+
+})
+
+var _ = Describe("{StoPoolExpMulPools}", func() {
+	/*
+		Steps : Verify if Multiple Pools exist in the system
+				Expand one of the available pools present from the list of available pools
+	*/
+	var testrailID = 51298
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51298
+	// testrailID Description : Having multiple pools and resize only one pool
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest("StoPoolExpMulPools", "Validate storage pool expansion using resize-disk option when multiple pools are present on the cluster", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	// Schedule the applications to run on K8s cluster
+	var contexts []*scheduler.Context
+	stepLog := "Has to schedule apps, and expand it by resizing a pool "
+	It(stepLog, func() {
+		log.InfoD(stepLog)
```

Comment:
Also for this, do we really need these 3 lines and printing same thing twice, can we just do this instead
```
It("Has to schedule apps, and expand it by resizing a pool") func() {
```
We don't need to print this `log.InfoD(stepLog)`, it will already get printed above by `It`

Reply:
Same thing for other tests you have

Reply:
not needed, I deleted the line

Reply:
Please make requested changes, I will review in the morning